js/js.tests/test/org/jetbrains/kotlin/js/test/BasicWasmBoxTest.kt

```diff
@@ -86,26 +111,32 @@ abstract class BasicWasmBoxTest(
     }
 
     private fun translateFiles(
+        testFile: File,
         units: List<TranslationUnit>,
         outputWatFile: File,
+        outputWasmFile: File,
         outputJsFile: File,
         config: JsConfig,
         testPackage: String?,
         testFunction: String
     ) {
         val filesToCompile = units.map { (it as TranslationUnit.SourceFile).file }
-        val debugMode = getBoolean("kotlin.js.debugMode")
+        val debugMode =getBoolean("kotlin.js.debugMode")
 
         val phaseConfig = if (debugMode) {
             val allPhasesSet = wasmPhases.toPhaseMap().values.toSet()
             val dumpOutputDir = File(outputWatFile.parent, outputWatFile.nameWithoutExtension + "-irdump")
             println("\n ------ Dumping phases to file://$dumpOutputDir")
+            println("\n ------  KT file://${testFile.absolutePath}")
+            println("\n ------ WAT file://$outputWatFile")
+            println("\n ------ WASM file://$outputWasmFile")
+            println(" ------  JS file://$outputJsFile")
             PhaseConfig(
                 wasmPhases,
                 dumpToDirectory = dumpOutputDir.path,
-                toDumpStateAfter = allPhasesSet,
-                toValidateStateAfter = allPhasesSet,
-                dumpOnlyFqName = null
+                // toDumpStateAfter = allPhasesSet,
```

Comment:
Have you forgotten to uncomment it?

Reply:
not really. this is a part which you would want to go and uncomment manually based on your debug needs.